site/installation/migration.md

```diff
@@ -193,21 +195,28 @@ CRD. Please refer to the detailed guide on [migrating inventory objects] to the
 
 ## Migration Steps
 
-Based on the changes discussed above, this section walks you through an end to
-end workflow to migrate an [example kpt package] which is compatible with
-`v0.39` version of kpt, and make it compatible with kpt `v1.0`.
+### For Package Publishers
 
-### Automated portion of migration
+Based on the changes discussed above, this section walks package publishers through an end to
+end workflow to migrate their existing packages which are compatible with
+`v0.39` version of kpt, and make them compatible with kpt `v1.0`.
 
-Invoke `gcr.io/kpt-fn/fix` function from the root directory of the target kpt
-package.
+#### Automated portion of migration
+
+Since you are the package publisher, you are expected to have the latest version
+of published package on your local. If you do not already have it, you can [git clone]
```

Comment:
I assume we are talking about local disk here?

Reply:
Yes. Updated.